partner_external_map/models/res_partner.py

```diff
@@ -26,6 +26,8 @@ def _address_as_string(self):
             addr.append(self.city)
         if self.state_id:
             addr.append(self.state_id.name)
+        if self.zip:
+            addr.append(self.zip)
```

Comment:
IMO this should be squashed on merge, so the commit comments are fairly irrelevant.